### ISSUE TITLE
Export default `toggleLink` behavior?

### ISSUE THREAD

**Issue by joshfrench** [bustlelabs/mobiledoc-kit#485]
What do you think about exposing the default `toggleLink` handler for use in editor toolkits? I'm prepping the React editor for release and realizing I'd rather just use the core `META+K` dialog on the `Link` button instead of supplying our own form there.

Happy to submit a pull if you're up for it, although I'm not sure where the right place to export it would be. Thoughts on where best to expose it?

**Comment by bantic** [comment]
@joshfrench Yeah, that certainly seems fine. +1 to code reuse.
Exporting a top-level "UI" might make sense for this. It would be good use this opportunity to change `toggleLink` to accept a configurable method for showing the prompt, i.e. pass it a callback-accepting function to use to show the prompt:
```
function toggleLink(editor, showPrompt=defaultShowPrompt) {
  ...
  let myPromptCallback = (url) => /* ... toggleMarkupWithUrl... */;
  showPrompt(message, defaultValue, myPromptCallback);
}

let defaultShowPrompt = (defaultValue, callback) => callback(window.prompt(message, defaultValue));
```

Exporting it as `Mobiledoc.UI.toggleLink` seems ok — what do you think?
If we do this we can also remove `Editor#showPrompt` as it is not used anywhere else.

**Comment by joshfrench** [comment]
Looks good to me, I'll put together a patch this week. Thanks!

**Comment by bantic** [comment]
closed by #491. thanks @joshfrench !
